src/views/OrderList.vue

```diff
@@ -252,8 +337,33 @@ button:not(.disabled) {
 .red {
     color: red;
 }
-/* 
-.item-row-notification {
-    display: flex;
-} */
+
+
+.edit-icon {
```

Comment:
Consider using `rem` instead of pixels for margins, paddings, etc.
`width` and `height` can still use pixel units but it is better to use `rem` if possible.

https://youtube.com/shorts/4iwQcvHYfWE?feature=share

Reply:
done!